### ISSUE TITLE
Nightly tests fail with "Cannot find TVM op config"

### ISSUE THREAD

**Issue by ptrendx** [apache/incubator-mxnet#16704]
## Description
Nightly tests on master branch fail with `Cannot find the TVM op config` message (look e.g. here: http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/NightlyTestsForBinaries/detail/master/493/pipeline)

### Error Message
```
======================================================================

ERROR: Failure: RuntimeError (Cannot find the TVM op config.

List of candidates:

/work/mxnet/python/mxnet/../../lib/tvmop.conf

/work/mxnet/python/mxnet/../../build/tvmop.conf)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/usr/local/lib/python2.7/dist-packages/nose/loader.py", line 418, in loadTestsFromName

    addr.filename, addr.module)

  File "/usr/local/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath

    return self.importFromDir(dir_path, fqname)

  File "/usr/local/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir

    mod = load_module(part_fqname, fh, filename, desc)

  File "/work/mxnet/tests/nightly/estimator/test_estimator_cnn.py", line 23, in <module>

    import mxnet as mx

  File "/work/mxnet/python/mxnet/__init__.py", line 90, in <module>

    from . import tvmop

  File "/work/mxnet/python/mxnet/tvmop.py", line 34, in <module>

    _CONF_TVM_OP = find_conf_path("tvmop")

  File "/work/mxnet/python/mxnet/libinfo.py", line 140, in find_conf_path

    'List of candidates:\n' + str('\n'.join(candidates_path)))

RuntimeError: Cannot find the TVM op config.

List of candidates:

/work/mxnet/python/mxnet/../../lib/tvmop.conf

/work/mxnet/python/mxnet/../../build/tvmop.conf
```

@apeforest @szha @reminisce

**Comment by szha** [comment]
@yzhliu

**Comment by hzfan** [comment]
Thank @ptrendx. I created a fix in #16773

**Comment by samskalicky** [comment]
@zachgk assign @szha

**Comment by ptrendx** [comment]
@hzfan @szha How can we test this fix without merging and waiting for the nightly tests?

**Comment by samskalicky** [comment]
@ptrendx #16773  is merged, can you cherry pick into 1.6?

**Comment by ptrendx** [comment]
Nightly tests did not run yet, are you sure it fixes the failures?

**Comment by ptrendx** [comment]
Fix merged to both master and 1.6, closing.
